ci-generate/src/toolchains/cargo.rs

```diff
@@ -32,9 +32,14 @@ pub(crate) fn cargo_config(config: Figment, matches: &ArgMatches) -> anyhow::Res
         || format!("{} description", &cargo.common.name),
         |desc| desc,
     );
-    let data = TemplateData::new(&cargo.common.project_path, &cargo.common.name)
-        .branch(&cargo.common.branch)
-        .license(&cargo.common.license);
+    let data = TemplateData::new(
+        &cargo.common.project_path,
+        &cargo.common.name,
+        &cargo.common.organization,
+        &cargo.common.repository,
```

Comment:
Can we insert all these values into a structure which will be shared among the methods? In this way we are going to reduce the number of function's parameters, even if we decide to add more parameters for the future. Let's adopt this behaviour everywhere